agentstack/_tools/py_sql/test.py

```diff
@@ -0,0 +1,66 @@
+from sql import Table
+from __init__ import construct_sql_query
+
+def test_query_construction():
```

Comment:
this is great for testing while building, but we dont want these files in the repo. eventually we'll have unit testing for tools but not at the moment

Reply:
a better option would be to add the psql tool to an existing example agent